blog/2023-11-06-public-addon-config.md

```diff
@@ -0,0 +1,52 @@
+---
+author: Mike Degatano
+authorURL: https://github.com/mdegat01
+title: "Public Addon Config"
+---
+
+Add-on's can now have a public folder for config or data files which users can see
+and modify but is still backed up with the add-on.
+
+Many add-ons ask users to provide files as part of addon configuration. Or generate
+files that they want users to be able to see and potentially modify. They typically
+handle this by including `config` and/or `share` in the list of folders to map.
+
+The problem with this is twofold:
+
+1. Nothing in `config` or `share` is backed up with the add-on. So backups of that add-on do not include all the necessary files to run it after restore.
+2. Add-ons which map `config` have far more access then they should since `config` includes all secrets and credentials used in your Home Assistant integrations.
```

Comment:
```suggestion
The problem with this is twofold:

1. Nothing in `config` or `share` is backed up with the add-on. So, backups of that add-on do not include all the necessary files to run it after restore.
2. Add-ons that map `config` have far more access than they should since `config` includes all secrets and credentials used in your Home Assistant integrations.
```